utils/cli/core/definition.py

```diff
@@ -162,6 +162,13 @@ def argparse(s):
     help='''number of seconds to wait until checking
             if data compression finished (necessary before uploading annotations)'''
 )
+task_create_parser.add_argument(
+    '--project_id',
```

Comment:
Let's name it as `--project`.

Reply:
the more verbose `project_id` helps disambiguate between project ID and project name